packages/nimble-blazor/NimbleBlazor/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ni/nimble-blazor",
+  "version": "16.0.4",
+  "description": "Blazor components for the NI Nimble Design System",
+  "scripts": {
+    "pack": "cross-env-shell dotnet pack -c Release -p:PackageVersion=$npm_package_version --output ../dist",
+    "invoke-publish": "npm run invoke-publish:setup && npm run invoke-publish:nuget && npm run invoke-publish:noop -- ",
+    "invoke-publish:setup": "rimraf --glob \"dist/NimbleBlazor.*.nupkg\" && npm run pack",
```

Comment:
You probably already did it, but JIC make sure to run the `invoke-publish:setup` commands locally to make sure they have the expected behavior in both packages, i.e. manually bump the version in the package.json and verify the output packages are using it.